lib/rubocop/cop/rails/reversible_migration.rb

```diff
@@ -7,16 +7,18 @@ module Rails
       # reversible.
       #
       # @example
+      #   # change_table
```

Comment:
There are examples for each special case (`drop_table`, `change_column_default`, `remove_column` and `remove_foreign_key`) so I added one entry for `change_table` as well. 

Maybe I could keep the general examples and only add the new ones at the bottom? In this case I'd still have to update the old examples since some uses of `change_table` are no longer "bad".